setup.sh

```diff
@@ -37,6 +37,7 @@ clear(){
 install(){
   echo 'Installing project dependencies...'
   npm install
+  npm install protractor@2.1.0 -g
```

Comment:
@contolini @anselmbradford well, there are few considerations that may or may not make it a good candidate for Travis. Protractor tests are end-2-end tests which means you need everything. Everything may be a pain in Travis. Alternatively, we can run it in sauce against some other instance, but that would lag behind by at least this commit, which sort  of defeats the purpose. There is one use case that @jimmynotjim  brought up where we can potentially leverage Travis for functional testing.